mLRS/Common/hal/esp-rxclock.h

```diff
@@ -106,11 +61,7 @@ class RxClockBase
     void Init(uint16_t period_ms);
     void SetPeriod(uint16_t period_ms);
     void Reset(void);
-
-    void init_isr_off(void);
-    void enable_isr(void);
-
-    uint16_t tim_10us(void);
+    void disable_isr(void);
```

Comment:
why not the same private initialized approach to attach/init omnly once?

Reply:
We still need to disable the timer for the eeprom activity in paramstore. Not an issue in setup() as the timeinterrupt hasn't been attached at this point.

Reply:
see comment in the above